Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -43,7 +46,6 @@ public struct RouteResponse {
 
 extension RouteResponse: Codable {
     enum CodingKeys: String, CodingKey {
-        case code
```

Comment:
I found this Key is unused.

Reply:
This key is [part of the Directions API response](https://docs.mapbox.com/api/navigation/directions/#response-retrieve-directions), but it looks like Directions is obtaining the `code` through ResponseDisposition:

https://github.com/mapbox/mapbox-directions-swift/blob/f46b02eeb196c4b23e96bc3f1a303266010e96ef/Sources/MapboxDirections/Directions.swift#L200-L215